watertap/unit_models/coag_floc_model.py

```diff
@@ -12,6 +12,7 @@
 ###############################################################################
 
 # Import Pyomo libraries
+from math import pi
```

Comment:
It ends up being the same thing, but we generally use `from idaes.core.util.constants import pi` for this (mostly just to advertize the IDAES Constants module).

Reply:
Made the suggested change